Project.toml

```diff
@@ -57,22 +63,22 @@ LegendHDF5IO = "0.1"
 LegendTextIO = "0.1"
 LsqFit = "0.13, 0.14, 0.15"
 MJDSigGen = "0.2"
-Parameters = "0.12"
+Parameters = "0.12, 0.13"
 Polynomials = "2, 3, 4"
 PropDicts = "0.2"
 RadiationDetectorDSP = "0.2.1"
 RadiationDetectorSignals = "0.3"
 RadiationSpectra = "0.5"
-Random123 = "1.2"
+Random123 = "1.2, 1.7" # Compatible with 1.7?
 RecipesBase = "1"
 Requires = "0.5, 1"
 SignalAnalysis = "0.4, 0.5, 0.6, 0.7, 0.8, 0.9"
-SolidStateDetectors = "0.10.3"
+SolidStateDetectors = "0.10.5"
 StaticArrays = "0.12, 1"
 Statistics = "1"
 StatsBase = "0.32, 0.33, 0.34"
 StructArrays = "0.5, 0.6"
 Tables = "1"
-TypedTables = "1.2"
+TypedTables = "1.2, 1.4" #Compatible with 1.4.6?
 Unitful = "1"
-julia = "1.10"
+julia = "1.11"
```

Comment:
This, I wouldn't do without reason.
Please also allow for the long-time stable version "1.10". 
There is no need to restrict this package to only 1.11. 
1.10 might be required to allow for the latest release of some packages, e.g. SolidStateDetectors